modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/search/ConfigurationModelIndexer.java

```diff
@@ -175,6 +155,8 @@ public Hits search(SearchContext searchContext) throws SearchException {
 
 			processHits(searchContext, hits);
 
+			_initialize();
```

Comment:
@Valmir-unicap What's the specific reason to put it here, instead of the beginning of this method? If there's any reason, could you reflect it in the commit message?

Reply:
@Valmir-unicap By the way, if the `_initialize()` invocation must be at that place, could you split the first commit into 2? Firstly, move the invocation of `initialize()`, then do inline? With this you can explain it separately.